CPAC/anat_preproc/anat_preproc.py

```diff
@@ -1128,37 +1128,15 @@ def freesurfer_abcd_brain_connector(wf, cfg, strat_pool, pipe_num, opt):
         name=f"wmparc_to_nifti_{pipe_num}",
     )
 
-    # Register wmparc file if ingressing FreeSurfer data
-    if strat_pool.check_rpool("pipeline-fs_xfm"):
-        wmparc_to_native = pe.Node(
-            Function(
-                input_names=["source_file", "target_file", "xfm", "out_file"],
-                output_names=["transformed_file"],
-                function=normalize_wmparc,
-            ),
-            name=f"wmparc_to_native_{pipe_num}",
-        )
-
-        wmparc_to_native.inputs.out_file = "wmparc_warped.mgz"
-
-        node, out = strat_pool.get_data("pipeline-fs_wmparc")
-        wf.connect(node, out, wmparc_to_native, "source_file")
-
-        node, out = strat_pool.get_data("pipeline-fs_raw-average")
-        wf.connect(node, out, wmparc_to_native, "target_file")
-
-        node, out = strat_pool.get_data("pipeline-fs_xfm")
-        wf.connect(node, out, wmparc_to_native, "xfm")
-
-        wf.connect(wmparc_to_native, "transformed_file", wmparc_to_nifti, "in_file")
-
-    else:
-        node, out = strat_pool.get_data("pipeline-fs_wmparc")
-        wf.connect(node, out, wmparc_to_nifti, "in_file")
+    node, out = strat_pool.get_data("pipeline-fs_wmparc")
+    wf.connect(node, out, wmparc_to_nifti, "in_file")
 
     wmparc_to_nifti.inputs.args = "-rt nearest"
 
-    node, out = strat_pool.get_data("desc-preproc_T1w")
+    if strat_pool.check_rpool("desc-restore_T1w"):
+        node, out = strat_pool.get_data("desc-restore_T1w")
+    else:
+        node, out = strat_pool.get_data("desc-preproc_T1w")
```

Comment:
I think these can just be

```suggestion
    node, out = strat_pool.get_data(["desc-restore_T1w", "desc-preproc_T1w"])
```

and be functionally the same